proto/cosmos/slashing/v1beta1/query.proto

```diff
@@ -10,50 +10,50 @@ option go_package = "github.com/cosmos/cosmos-sdk/x/slashing/types";
 
 // Query provides defines the gRPC querier service
 service Query {
```

Comment:
note to contributors who work on proto schemas, let's please stick to a sane formatting config 🙏

Reply:
Can we use editorConfig in sdk to ensure this?

Reply:
yes, good idea. I use vscode. I need to figure out how to do this. If you know, please share :)

Reply:
@webmaster128 had a suggestion on how to do this

Reply:
https://editorconfig.org/ can serve as a baseline text formatter, taking care of indent, line ends, trailing whilespace, encoding and so on. But it is not a .proto specific formatter.

Reply:
As an IntelliJ/goland user, I would like some format script I can run without needing vscode

Reply:
You don't need vscode -- vscode just runs the standard clang-formatter, which I presume is the exact same thing goland would do. In the root of this directory, we could have vscode and goland settings files.